apps/sports-helsinki/src/domain/search/searchHeader/searchHeader.module.scss

```diff
@@ -50,31 +49,35 @@
     background-color: var(--background-color);
     display: flex;
     flex-direction: row;
-    align-items: center;
+    align-items: auto;
     position: fixed;
     bottom: 0;
     left: 0;
     right: 0;
+    padding: 0 0 $spacing-layout-xs 0;
```

Comment:
Is there a real design reasoning why this xs-padding is set to the bottom? Because of that, there is a transparent line under the button-row:

<img width="432" alt="image" src="https://user-images.githubusercontent.com/389204/228857065-969a83e1-5c6f-4d59-aa27-4c7a71b9cb4d.png">


```suggestion
```